docs/FAQ.md

```diff
@@ -82,6 +82,30 @@ az acr repository show-manifests -n myRegistry --repository myRepository --query
 
 Note: You can add `-y` in the delete command to skip confirmation
 
+## I deleted images to free up registry quota usage but the available quota doesn't get updated
+
+This can happen if the underlying layers are still being referenced by other container images. If you delete an image with no references, the registry usage will be updated after a few minutes. Here are the steps for verifying the quota usage.
+
+        Download a test image from DockerHub
```

Comment:
should fix the formatting, it's garbled; use `sh` or similar as syntax hightlighting for the code blocks

Reply:
Avoid indenting blocks here.

Reply:
Thanks. Edited in VS to fix the formatting.